src/reducers/utils.js

```diff
@@ -0,0 +1,19 @@
+export function compareAisle(a, b) {
```

Comment:
Unfortunately, your compare aisle doesn't sort numerical items properly. Aisles are stored/sent as strings (as you can have aisles like "dairy" or "produce"). Right now, you are doing string comparisons---you end up with sorts like this:
![localhost_3000_lists_5c40df593fe04f89a378b3fc](https://user-images.githubusercontent.com/164159/51575878-8dc6c180-1e79-11e9-8edc-43ff2d017865.png)

We also probably want the comparison function do something sane for things that aren't numbers.